bootstrap.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 set -eu
 
-## Make sure we'll use ruby 2.6
-amazon-linux-extras install -y ruby2.6
+## Make sure we have Ruby 3 installed directly on the instance
+amazon-linux-extras install -y ruby3.0
```

Comment:
The [previous upgrade attempt](https://github.com/panorama-ed/buildkite-assets/pull/199/files#diff-99d5a2a8ca4290b61836b3ad37941f2b925e9ff3151c90dcd5ea82364e4122a1R4) tried to use `ruby3.1` here, but that package doesn't exist. This was causing build agents to crash on boot:
```
<13>Dec  5 16:18:18 user-data: Topic ruby3.1 is not found.
<13>Dec  5 16:18:18 user-data: ++ on_error 206
<13>Dec  5 16:18:18 user-data: ++ local exitCode=4
<13>Dec  5 16:18:18 user-data: ++ local errorLine=206
<13>Dec  5 16:18:18 user-data: ++ [[ 4 != 0 ]]
<13>Dec  5 16:18:18 user-data: ++ aws autoscaling set-instance-health --instance-id i-02b500f2e8f001dfc --health-status Unhealthy
<13>Dec  5 16:18:19 user-data: ++ /opt/aws/bin/cfn-signal --region us-east-1 --stack buildkite-version-5-10-0 --reason 'Error on line 206: +++ tail -n 1 /var/log/elastic-stack.log' --resource AgentAutoScaleGroup --exit-code 4
<13>Dec  5 16:18:21 user-data: ValidationError: Stack arn:aws:cloudformation:us-east-1:144262447452:stack/buildkite-version-5-10-0/e3a89e40-06d6-11ed-9b90-0ece9efc3205 is in CREATE_COMPLETE state and cannot be signaled
[   30.347682] cloud-init[3616]: Dec 05 16:18:21 cloud-init[3616]: util.py[WARNING]: Failed running /var/lib/cloud/instance/scripts/part-003 [1]
[   30.350678] cloud-init[3616]: Dec 05 16:18:21 cloud-init[3616]: cc_scripts_user.py[WARNING]: Failed to run module scripts-user (scripts in /var/lib/cloud/instance/scripts)
[   30.350863] cloud-init[3616]: Dec 05 16:18:21 cloud-init[3616]: util.py[WARNING]: Running module scripts-user (<module 'cloudinit.config.cc_scripts_user' from '/usr/lib/python2.7/site-packages/cloudinit/config/cc_scripts_user.pyc'>) failed
[   30.411840] cloud-init[3616]: Cloud-init v. 19.3-45.amzn2 finished at Mon, 05 Dec 2022 16:18:21 +0000. Datasource DataSourceEc2.  Up 30.40 seconds
[[1;31mFAILED[0m] Failed to start Execute cloud user/final scripts.
See 'systemctl status cloud-final.service' for details.
```

I verified that `ruby3.0` is available from `amazon-linux-extras` by connecting to an existing instance:
```
# amazon-linux-extras list | grep ruby
 39  ruby2.6=latest           enabled      [ =2.6  =stable ]
 57  ruby3.0                  available    [ =stable ]
```

Ultimately, this version is only used for a handful of agent setup commands that run directly on the instance. Moving it to 3.0 instead of 3.1 is fine. `panorama_command.sh` does its work inside `Docker`; this Ruby here is only used by commands that run directly on the instance, and I'm not sure any of those in our pipeline actually use Ruby at all.